From 19d6be8095e87682b04a9fb69ee8f6fd37e16a50 Mon Sep 17 00:00:00 2001
From: Waldemar Brodkorb <wbx@openadk.org>
Date: Wed, 20 Aug 2014 20:32:38 +0200
Subject: [PATCH 06/11] do not depend on compile

When you cross-compile the testsuite and then try to run
on a system without gcc/binutils it would be nice if no compiling
or linking is tried.

Signed-off-by: Waldemar Brodkorb <wbx@uclibc-ng.org>
---
 test/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/Makefile b/test/Makefile
index b759bf9..787c530 100644
--- a/test/Makefile
+++ b/test/Makefile
@@ -45,7 +45,7 @@ endif
 
 test check all: run
 
-run: compile subdirs_run
+run: subdirs_run
 
 compile: $(top_builddir)$(LOCAL_INSTALL_PATH) subdirs_compile
 
-- 
1.8.5.2 (Apple Git-48)

